app/code/Piimega/Maksuturva/i18n/fi_FI.csv

```diff
@@ -1,35 +1,33 @@
-"Unknown error on maksuturva payment module.", "Tuntematon virhe Maksuturva moduuli."
-"Error on Maksuturva return", "Virhe Maksuturva paluu"
+"Manually Query Maksuturva API For Orders Created In Past 2 Hours", "Synkronoi tilaukset (luotu 2:n tunnin sisällä)"
+"Manually Query Maksuturva API For Orders Created In Past Week", "Synkronoi tilaukset (luotu viikon sisällä)"
+"Unknown error on maksuturva payment module.", "Tapahtui tuntematon virhe."
+"Payment canceled on Maksuturva", "Maksu peruttu"
+"You have cancelled your payment on Maksuturva.", "Olet peruuttanut maksun."
+"Unable cancel order that is already paid.", "Tilausta ei voitu peruuttaa koska se on jo maksettu."
+"Waiting for delayed payment confirmation from Maksuturva", "Odottaa vahvistusta"
+"Maksuturva returned an error on your payment.", "Maksu epäonnistui."
+"Invalid hash returned", "Tarkiste on viallinen"
+"Gateway returned an empty field", "Pakollinen tieto puuttuu"
+"Value returned from Maksuturva does not match:", "Arvo ei täsmää:"
+"Shipping and payment costs returned from Maksuturva do not match.", "Maksuturvan palauttamat toimitus- ja maksukulut eivät täsmää verkkokauppatilauksen kanssa."
+"Error on Maksuturva return", "Tapahtui virhe palatessa verkkokauppaan"
 "Choose payment method…", "Valitse maksutapa…"
-"Shipping", "Laivaus"
-"Payment fee", "Maksu hinta"
-"Authorization identified by Maksuturva", "Valtuutus tunnistetaan Maksuturva"
-"Payment identified by Maksuturva. Invoice saved.", "Maksu tunnistaa Maksuturva. Lasku tallennettu."
-"Payment identified by Maksuturva", "Maksu tunnistaa Maksuturva"
-"Payment identified by Maksuturva. invoices already exist", "Maksu tunnistaa Maksuturva. laskut jo olemassa"
-"Payment canceled by Maksuturva", "Maksu peruutettu Maksuturvalla"
-"No change, still awaiting payment", "Ei muutosta, odottavat yhä maksu"
-"You will be redirected to the Maksuturva website in a few seconds.", "Sinut ohjataan Maksuturvan verkkosivuilla muutamassa sekunnissa."
-"Capture action is not available.", "Capture toiminto ei ole käytettävissä."
-"Run Query Maksuturva API for orders missing payments manually success", "Suorita Query Maksuturva API Tilaukset jätetyistä maksuista käsin menestys"
-"Unknown exception happen.", "Tuntematon poikkeus tapahtuu."
-"Your order is not valid or is already paid.", "Tilauksesi ei ole voimassa tai on jo maksettu."
-"Payment %1 by Maksuturva. NOTE: Change in the sellercosts + %2 EUR.", "Maksu% 1 Maksuturvalla. HUOM: Muutos sellercosts +% 2 euroa."
-"Payment 1% by Maksuturva. NOTE: Change in the sellercosts %2 EUR.", "Maksu 1% Maksuturvalla. HUOM: Muutos sellercosts% 2 euroa."
-"Payment %1 by Maksuturva", "Maksu% 1 Maksuturvalla"
\ No newline at end of file
+"Shipping", "Toimitus"
+"Payment fee", "Maksukulu"
+"Authorization identified by Maksuturva", "Maksutapahtuma valtuutettu"
+"Payment identified by Maksuturva. Invoice saved.", "Lasku on tallennettu."
+"Payment identified by Maksuturva", "Maksuturva on käsitellyt tapahtuman"
```

Comment:
Can maybe be changed to just "Maksu onnistui"?

Reply:
Discussion moved to Slack.